packages/react/src/select/popup/useSelectPopup.ts

```diff
@@ -298,15 +298,14 @@ export function useSelectPopup(): useSelectPopup.ReturnValue {
 
           handleScrollArrowVisibility();
         },
-        style: {
-          ...(alignOptionToTrigger && {
+        ...(alignOptionToTrigger && {
+          style: {
             position: 'relative',
             maxHeight: '100%',
             overflowX: 'hidden',
             overflowY: 'auto',
-          }),
-          outline: '0',
-        },
```

Comment:
Is there a need for changing `outline` though? Radix sets it by default and it's kind of annoying to see as a default.

Reply:
Yeah I wanted to talk through this:

I'm using `outline` for outer borders and this inline style overrides my CSS. The other styles here also seem a little bit opinionated, so probably the best solution is to render this in a style tag under `:where(#popup-id)` to have the best of the both worlds.

We are also being a little inconsistent with the outline handling since dialog popup outlines aren't removed at the moment.

Reply:
Also I saw an outline on Positioner _sometimes_ popping up in Safari? That was a real surprise